site/gatsby-site/src/components/submissions/SubmissionWizard.js

```diff
@@ -78,13 +91,23 @@ const SubmissionWizard = ({ submitForm, initialValues, urlFromQueryString }) =>
 
         const cloudinary_id = getCloudinaryPublicID(news.image_url);
 
-        const newValues = {
+        let newValues = {
```

Comment:
Since the variable is not being reassigned, it is appropriate to retain the `const` keyword.